low-level/files/opt/seagate/sspl/conf/sspl.conf.LDR_R2.yaml

```diff
@@ -0,0 +1,218 @@
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License along
+# with this program. If not, see <https://www.gnu.org/licenses/>. For any questions
+# about this software or licensing, please email opensource@seagate.com or
+# cortx-questions@seagate.com.
+
+# Version 2.0.0
+
+# TODO: Rename limit_consul_memory to limit_persistent_memory_use
+# and move it to DATASTORE
+
+SSPL_LL_SETTING:
+   core_processors:
+      - RabbitMQegressProcessor
+      - RabbitMQingressProcessor
+      - LoggingProcessor
+   message_handlers:
+      - DiskMsgHandler
+      - LoggingMsgHandler
+      - ServiceMsgHandler
+      - NodeDataMsgHandler
+      - NodeControllerMsgHandler
+      - RealStorEnclMsgHandler
+      - RealStorActuatorMsgHandler
+   sensors:
+      - ServiceWatchdog
+      - RAIDsensor
+      - NodeData
+      - RealStorFanSensor
+      - RealStorPSUSensor
+      - RealStorControllerSensor
+      - RealStorDiskSensor
+      - RealStorSideplaneExpanderSensor
+      - RealStorLogicalVolumeSensor
+      - IEMSensor
+      - NodeHWsensor
+      - RealStorEnclosureSensor
+      - SASPortSensor
+      - MemFaultSensor
+      - CPUFaultSensor
+   actuators:
+      - Service
+      - RAIDactuator
+      - Smartctl
+      - NodeHWactuator
+      - RealStorActuator
+   degraded_state_modules:
+      - ServiceWatchdog
+      - RAIDsensor
+      - NodeData
+      - IEMSensor
+      - NodeHWsensor
+      - DiskMsgHandler
+      - LoggingMsgHandler
+      - ServiceMsgHandler
+      - NodeDataMsgHandler
+      - NodeControllerMsgHandler
+      - RealStorActuatorMsgHandler
+      - SASPortSensor
+      - MemFaultSensor
+      - CPUFaultSensor
+
+RABBITMQINGRESSPROCESSOR:
+   virtual_host: SSPL
+   queue_name: actuator-req-queue
+   exchange_name: sspl-in
+   routing_key: actuator-req-key
+   username: sspluser
+   password: gAAAAABgB8tRJLZBfI6uCAtt4B8YYjveDvhHnFCbeKYsXuYxSLl3KZE5Tp94Jl2qQq9-X7x7_CinZZSh4cm9rMJ4G3HKz36I-Q==
```

Comment:
let's set a blank password here.

Reply:
Can you recommend a way to get password ?

Reply:
@thavanathan default password is fine, so dev setup is fast.
This field needs manual update to sspl.conf with updated password, if changed from default.